### DISCUSSION TITLE
Fresh Install issues, no fleetdm

### DISCUSSION THREAD

**Discussion by tbaror** [Security-Onion-Solutions/securityonion#5638] [General]
Hello ,
Just installed new SO server distributed mode on Ubuntu server, the installation finished with error as follows  shown below 
When i get into fleetdm page i get `404 page not found` although so-status ahows its running
any idea?
please advice.
Thanks

errors
`
Line 2646: *** Restarting Salt to fix any SSL errors. *** Line 8837: +log_level: error Line 8838: +log_level_logfile: error Line 9145: +log_level: error Line 9146: +log_level_logfile: error Line 18610: Error response from daemon: Container ea6c46f83367f588e036f03ddc56738783c2d3d4c83ff62745dc7158bcb923a3 is not running Line 18611: Error response from daemon: Container ea6c46f83367f588e036f03ddc56738783c2d3d4c83ff62745dc7158bcb923a3 is not running Line 18612: Error response from daemon: Container ea6c46f83367f588e036f03ddc56738783c2d3d4c83ff62745dc7158bcb923a3 is not running Line 18613: Error response from daemon: Container ea6c46f83367f588e036f03ddc56738783c2d3d4c83ff62745dc7158bcb923a3 is not running Line 18614: Error response from daemon: Container ea6c46f83367f588e036f03ddc56738783c2d3d4c83ff62745dc7158bcb923a3 is not running Line 18615: Error response from daemon: Container ea6c46f83367f588e036f03ddc56738783c2d3d4c83ff62745dc7158bcb923a3 is not running Line 18617: [ERROR ] Command 'docker' failed with return code: 1 Line 18618: [ERROR ] stdout: Error response from daemon: Container ea6c46f83367f588e036f03ddc56738783c2d3d4c83ff62745dc7158bcb923a3 is not running Line 18618: [ERROR ] stdout: Error response from daemon: Container ea6c46f83367f588e036f03ddc56738783c2d3d4c83ff62745dc7158bcb923a3 is not running Line 18619: [ERROR ] retcode: 1 Line 18620: [ERROR ] Command 'docker' failed with return code: 1 Line 18621: [ERROR ] output: Error response from daemon: Container ea6c46f83367f588e036f03ddc56738783c2d3d4c83ff62745dc7158bcb923a3 is not running Line 18621: [ERROR ] output: Error response from daemon: Container ea6c46f83367f588e036f03ddc56738783c2d3d4c83ff62745dc7158bcb923a3 is not running Line 19909: [INFO ] {'pid': 41614, 'retcode': 0, 'stdout': 'Waiting for value \'"status":"green"\' at \'http://1.2.3.4:9400/_cluster/health\' (1/300)\nServer is not ready\nWaiting for value \'"status":"green"\' at \'http://1.2.3.4:9400/_cluster/health\' (2/300)\nServer is not ready\nWaiting for value \'"status":"green"\' at \'http://1.2.3.4:9400/_cluster/health\' (3/300)\nServer is not ready\nWaiting for value \'"status":"green"\' at \'http://1.2.3.4:9400/_cluster/health\' (4/300)\nServer is not ready\nWaiting for value \'"status":"green"\' at \'http://1.2.3.4:9400/_cluster/health\' (5/300)\nServer is not ready\nWaiting for value \'"status":"green"\' at \'http://1.2.3.4:9400/_cluster/health\' (6/300)\nServer is not ready\nWaiting for value \'"status":"green"\' at \'http://1.2.3.4:9400/_cluster/health\' (7/300)\nServer is not ready\nWaiting for value \'"status":"green"\' at \'http://1.2.3.4:9400/_cluster/health\' (8/300)\nServer is not ready\nWaiting for value \'"status":"green"\' at \'http://1.2.3.4:9400/_cluster/health\' (9/300)\nServer is not ready\nWaiting for value \'"status":"green"\' at \'http://1.2.3.4:9400/_cluster/health\' (10/300)\nReceived expected response; proceeding.\nWaiting for value \'Cortex\' at \'http://1.2.3.4:9001/cortex/\' (1/300)\nReceived expected response; proceeding.\n{"_id":"person@place.invalid","createdBy":"init","createdAt":1632402163878,"name":"person@place.invalid","roles":["superadmin"],"organization":"cortex","status":"Ok","_type":"user","_parent":null,"_routing":"person@place.invalid","_seqNo":0,"_primaryTerm":1,"id":"person@place.invalid","hasKey":true,"hasPassword":false}{"status":"Active","createdBy":"person@place.invalid","_id":"SecurityOnion","createdAt":1632402165628,"description":"SecurityOnion organization created by Security Onion setup","_type":"organization","_parent":null,"_routing":"SecurityOnion","_seqNo":0,"_primaryTerm":1,"id":"SecurityOnion","name":"SecurityOnion"}{"createdBy":"person@place.invalid","name ... Line 20303: {"_id":"person@place.invalid","createdBy":"init","createdAt":1632402163878,"name":"person@place.invalid","roles":["superadmin"],"organization":"cortex","status":"Ok","_type":"user","_parent":null,"_routing":"person@place.invalid","_seqNo":0,"_primaryTerm":1,"id":"person@place.invalid","hasKey":true,"hasPassword":false}{"status":"Active","createdBy":"person@place.invalid","_id":"SecurityOnion","createdAt":1632402165628,"description":"SecurityOnion organization created by Security Onion setup","_type":"organization","_parent":null,"_routing":"SecurityOnion","_seqNo":0,"_primaryTerm":1,"id":"SecurityOnion","name":"SecurityOnion"}{"createdBy":"person@place.invalid","name":"soadmin","organization":"SecurityOnion","_id":"soadmin","createdAt":1632402166580,"roles":["read","analyze","orgadmin"],"status":"Ok","_type":"user","_parent":null,"_routing":"soadmin","_seqNo":1,"_primaryTerm":1,"id":"soadmin","hasKey":true,"hasPassword":false}{"type":"NotFoundError","message":"Worker Urlscan_io_Search_0_1_0 not found"}{"version":"2.0","workerDefinitionId":"CERTatPassiveDNS_2_0","createdBy":"soadmin","license":"AGPL-V3","dataTypeList":["domain","fqdn","ip"],"createdAt":1632402167666,"author":"Nils Kuhnert, CERT-Bund","baseConfig":"CERTatPassiveDNS","name":"CERTatPassiveDNS_2_0","type":"analyzer","_id":"4f28afc20d78f98df425e36e561af33f","url":"https://github.com/BSI-CERT-Bund/cortex-analyzers","description":"Checks CERT.at Passive DNS for a given domain.","command":"/Cortex-Analyzers/analyzers/CERTatPassiveDNS/certat_passivedns.py","_type":"worker","_parent":"SecurityOnion","_routing":"SecurityOnion","_seqNo":2,"_primaryTerm":1,"id":"4f28afc20d78f98df425e36e561af33f","analyzerDefinitionId":"CERTatPassiveDNS_2_0"} Line 32590: Errors detected during setup; skipping post-setup steps to allow for analysis of failures.
`

(edited to remove private IP/email address data)

**Comment by defensivedepth** [comment]
Can you please post some additional technical details? See `Provide sufficient technical info` on https://github.com/Security-Onion-Solutions/securityonion/discussions/1720
